sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -36,31 +34,40 @@ let envVars: any;
 /**
  * Utility to return cached map of environment variables,
  * or create and return one from configured values if not existing.
+ *
+ * The utility helps use the right environment variable key based on targetted platform and type.
+ * Specifically, we use different Service Bus namespaces for browser Vs node test runs.
+ * Thus, the connection string value is retrieved from `SERVICE_BUS_CONNECTION_STRING_BROWSER`
+ * environment variable key for browser, and from `SERVICE_BUS_CONNECTION_STRING` for Node.
  */
-export function getEnvVars(): { [key in EnvVarKeys]: any } {
+export function getEnvVars(): { [key in EnvVarNames]: any } {
   if (envVars != undefined) {
     return envVars;
   }
 
+  let serviceBusConnectionStringEnvVarKey: string = EnvVarNames.SERVICEBUS_CONNECTION_STRING.valueOf();
```

Comment:
Perhaps rename this to `serviceBusConnectionStringEnvVarName` based on @bterlson's feedback?